main/templates/bit/script.html

```diff
@@ -1,6 +1,6 @@
 # if config.DEVELOPMENT
-  <script src="/p/dev/script/ext.js?{{config.CURRENT_VERSION_ID}}"></script>
-  <script src="/p/dev/script/script.js?{{config.CURRENT_VERSION_ID}}"></script>
+  <script src="/p/dev/script/ext.js?{{random()}}"></script>
+  <script src="/p/dev/script/script.js?{{random()}}"></script>
```

Comment:
I think app engine changed something so it wouldn't update the local version ID on every refresh..

Reply:
hmm, that could have some side effects... I would rather like to have this predictable

Reply:
for localhost..? does it really matter?

Reply:
Cannot (yet) reproduce that the ``{{config.CURRENT_VERSION_ID}}`` approach doesn't work.

Reply:
try to change only js/css and check if the change is actually happening with autorefresh

Reply:
You mean on localhost with the development server?